config/initializers/task.rb

```diff
@@ -1,18 +1,18 @@
 # Override Rake::Task to include global exception handling
-
-binding.pry
 if !Rails.env.test?
-  module Rake
-    class Task
-      alias :orig_execute :execute
-      def execute(args=nil)
-        orig_execute(args)
-      rescue Exception => ex
-        binding.pry
-        raise ex if [Interrupt, SystemExit, SignalException].include?(ex.class)
-        Raygun.track_exception(ex)
-        raise ex
+  begin
+    module Rake
+      class Task
+        alias :orig_execute :execute
+        def execute(args=nil)
+          orig_execute(args)
+        rescue Exception => ex
+          raise ex if [Interrupt, SystemExit, SignalException].include?(ex.class)
+          Raygun.track_exception(ex)
+          raise ex
+        end
       end
     end
+  rescue
```

Comment:
remove pry (not sure how rubo didn't catch)
wrap in try/rescue for when we're running outside of a rake task

Reply:
try catch is begin rescue in ruby right?

Reply:
yes being rescue!